### PR TITLE
change tubes to display group_ids instead of oligo ids

### LINKED ISSUE #349: GPL-nnn show tag name not sequence of tag in run creation
**User story**
As a Traction user, Craig, I would like the libraries list in the run creation page to to contain the tag name of the library, not the tag sequence as the names are human readable and the sequence is not

**Who the primary contacts are for this story**
Craig Corton

**Acceptance criteria**
To be considered successful the solution must allow:
1 - In the Run creation page, the libraries rows contain the readable tag name(s) per sample(s)

**Additional context**
See RT ticket #684947

### PR DESCRIPTION
**Pull request by BenTopping**
closes #349

### CODE REVIEW

**Comment by BenTopping** [comment]
WIP

File: tests/unit/components/Tube.spec.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ describe('Tube.vue', () => {
   let tube, wrapper, props
 
   beforeEach(() => {
-    props = { barcode: 'TRAC-1', sample_names: 'Sample1,Sample2', tag_oligos: 'ATGC,CGAT' }
+    props = { barcode: 'TRAC-1', sample_names: 'Sample1,Sample2', tag_group_ids: '1' }
```
Comment (harrietc52) [inline tests/unit/components/Tube.spec.js]: Here tag_group_ids would be '1,2' for example

File: tests/data/StoreLibraries.json [hunk 2]
```diff
@@ -11,7 +11,6 @@
     "created_at": "09/23/2019 11:18",
     "deactivated_at": null,
     "sample_names": "4616STDY7535900,4616STDY7535900",
-    "tag_oligos": "ATGC,CGTA",
     "tag_group_ids": "1"
```
Comment (harrietc52) [inline tests/data/StoreLibraries.json]: Here tag_group_ids would be "1,2" I guess?

Comment (BenTopping) [reply tests/data/StoreLibraries.json]: yep, good spot

**Pull request merged**